lib_oauth2/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
What is index.js for?

Reply:
Nothing!  Tossing.

Reply:
A reminder of a misguded effort.  Removing.